spatial-tracking-explainer.md

```diff
@@ -197,37 +197,58 @@ function onSessionStarted(session) {
+However, once a pose is initially established for a viewer or input source, pose matrices should continue to be provided even during tracking loss. The `emulatedPosition` attribute of `XRPose` indicates that the position component of the retrieved pose matrix does not represent an actively tracked position. There are a number of reasons this might be the case. For example:
+* A viewer with orientation-only tracking, whose position within an `eye-level` reference space represents neck modeling.
+* A viewer that has temporarily lost positional tracking, whose position within a reference space represents the viewer's last-known position in that space, plus inertial dead reckoning and/or neck modeling to continue providing a position.
+* A motion controller with orientation-only tracking, which is positioned at an assumed position, e.g. by the user's hip.
+* A motion controller that has temporarily lost positional tracking but is still held, whose orientation continues to update at the last-known position relative to the viewer.
 
-### Viewer space
-Calls to `XRFrame.getViewerPose()` return an `XRViewerPose` object which contains the pose of the viewer along with the views to be rendered. Sometimes it is useful to have access to the `XRSpace` represented by the viewer directly, such when the developer wants to use it to compare locations against other `XRSpace` objects.
+This base tracking loss behavior enables developers to then build whatever tracking loss behavior their scenario requires.
 
-```js
-  let pose = xrFrame.getPose(preferredInputSource.gripSpace, xrSession.viewerSpace);
-  if (pose) {
-    // Calculate how far the motion controller is from the user's head
-  }
-```
+For viewer poses, it's common in VR experiences to let the world drag along with you during positional tracking loss, as this can be preferable to halting the experience. By continuing to render the experience with viewer poses even when `emulatedPosition` is true, this behavior can be obtained easily. In contrast, when building an AR experience where anchoring of rendered objects to the real world is more critical, developers can detect that the viewer pose's `emulatedPosition` has become false and stop rendering the main scene, falling back to an orientation-only warning.
+
+For input source poses, motion controllers may have their own inertial tracking that can continue updating orientation while positional tracking is lost. If using a controller primarily to target distant objects, developers may simply ignore `emulatedPosition` and continue to point using each frame's updated target ray as the user rotates the controller. If using a controller to do fine operations, such as painting at the controller's tip, developers may instead choose to stop painting when `emulatedPosition` becomes false, ensuring that only high-quality paint strokes are drawn.
+
+#### Tracking recovery
```

Comment:
Like much of the explainer, this section both describes the API behavior that a developer "must" or "should" be able to count on from a WebXR UA, along with developer-focused guidance on how they can then best build an app on top of that behavior.

I do see that this section maybe has more "best practice" content than most, but it's woven in with the key paragraph that establishes the common situations where a UA "should" handle tracking loss in a specific way:

> However, once a pose is initially established for a space (e.g. a `viewer` reference space or an input source's `gripSpace`), pose matrices should continue to be provided even during tracking loss. The `emulatedPosition` attribute of `XRPose` indicates that the position component of the retrieved pose matrix does not represent an actively tracked position. There are a number of reasons this might be the case. For example:
> * A viewer with orientation-only tracking, whose position within a `local` reference space represents neck modeling.
> * A viewer that has temporarily lost positional tracking, whose position within a reference space represents the viewer's last-known position in that space, plus inertial dead reckoning and/or neck modeling to continue providing a position.
> * A motion controller with orientation-only tracking, which is positioned at an assumed position, e.g. by the user's hip.
> * A motion controller that has temporarily lost positional tracking but is still held, whose orientation continues to update at the last-known position relative to the viewer.

Do you believe we should more explicitly separate that paragraph from the guidance that follows?

Reply:
Also, I tweaked the first sentence to say "primary reference space" rather than "`XRReferenceSpace`" to hopefully make this example feel less like a firm tie to that exact type.

Reply:
Thinking about this more... maybe all it needs is subheadings to separate out the different sections of best practices?  WDYT?